app/Helpers/database/game.php

```diff
@@ -977,6 +981,16 @@ function getRichPresencePatch(int $gameId, ?string &$dataOut): bool
 
 function GetPatchData(int $gameID, ?User $user, int $flag): array
```

Comment:
Note that these `GetPatchData()` changes will cause a large merge conflict in https://github.com/RetroAchievements/RAWeb/pull/2857 (that PR removes `GetPatchData()` entirely).

No action needs to be taken here. I am just leaving this comment to raise awareness that this will happen. I am happy to resolve the conflicts in #2857.

The new tests in `PatchDataTest` will still be compatible with the changes in #2857.